apk-generator/v2/app/generator/generator.py

```diff
@@ -221,6 +221,7 @@ def download_event_data(self):
         self.save_file_in_temp_assets('speakers')
         self.save_file_in_temp_assets('sponsors')
         self.save_file_in_temp_assets('tracks')
+	self.save_file_in_temp_assets('session_types')
```

Comment:
Still indentation error. You have used a tab, whereas the file is indented using 4 spaces